### PR TITLE
Added comments to describe graph overlay with video

### PR DESCRIPTION
**Pull request by DerenDesign**
I will later try to have both of the graphs to become combined and position the video in a way that is easy to interpret.

### CODE REVIEW

File: templates/graph.html [hunk 1]
```diff
@@ -4,105 +4,176 @@
...
+            duration: 0 //transitions right away
+        },
+        frame: {
+            duration: 0, // Instantly change to the new frame without any animation effect
+            redraw: false
+        }
+        });
+    }
+
+    function animatePlot(videoTime) { //videotime describes the time in the video that is input into the program
```
Comment (allison-lane) [inline templates/graph.html:133]: duplicate function?

Comment (DerenDesign) [reply templates/graph.html:133]: "plot" is the x-direction force and "plot2" is the y-direction force. I will combine them and orientate them better with the video in a later commit.

Comment (allison-lane) [reply templates/graph.html:133]: Oh I see did you intend to give them different names? I noticed in lines 171 you call the function `animatePlot2`

Comment (DerenDesign) [reply templates/graph.html:133]: Yes, those two different animate plot and animateplot2 are for the two different graphs.

Comment (allison-lane) [reply templates/graph.html:133]: Right now they have the same name, both functions are called `animatePlot`

**Pull request closed without merging**